packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -180,51 +182,80 @@ export const SubscriptionCreate = ({
             </Localized>
           </div>
 
-          {!hasExistingCard(customer) && paypalScriptLoaded && (
-            <Suspense fallback={<div>Loading...</div>}>
-              <PaypalButton
-                apiClientOverrides={apiClientOverrides}
-                customer={customer}
-                setPaymentError={setPaymentError}
-                idempotencyKey={submitNonce}
-                ButtonBase={paypalButtonBase}
-                currencyCode={selectedPlan.currency}
-              />
-            </Suspense>
-          )}
-
-          <h3 className="billing-title">
-            <Localized id="sub-update-title">
-              <span className="title">Billing Information</span>
-            </Localized>
-          </h3>
+          <div className="subscription-create-pay-with-other">
+            {!hasExistingCard(customer) && paypalScriptLoaded && (
+              <Suspense fallback={<div>Loading...</div>}>
+                <Localized id="pay-with-heading-other">
+                  <p className="pay-with-heading">Select payment option</p>
+                </Localized>
+                <div className="paypal-button">
+                  <PaypalButton
+                    apiClientOverrides={apiClientOverrides}
+                    customer={customer}
+                    setPaymentError={setPaymentError}
+                    idempotencyKey={submitNonce}
+                    ButtonBase={paypalButtonBase}
+                    currencyCode={selectedPlan.currency}
+                  />
+                </div>
+              </Suspense>
+            )}
+          </div>
 
-          {!hasExistingCard(customer) && <AcceptedCards />}
+          <div className="subscription-create-pay-with-card">
+            {!hasExistingCard(customer) && !paypalScriptLoaded && (
+              <div>
+                <Localized id="pay-with-heading-card-only">
```

Comment:
`pay-with-heading-card-only` and `pay-with-heading-card-or` are missing in the .ftl

Reply:
Resolved.